src/components/Dashboard/Alerts/Amalgamation.vue

```diff
@@ -1,51 +1,46 @@
 <template>
-  <v-expansion-panels
-    id="amalgamation-container"
-    v-model="panel"
-  >
-    <v-expansion-panel class="mb-6">
-      <v-expansion-panel-header
-        hide-actions
-        class="d-flex justify-space-between px-6 py-5"
+  <v-expansion-panel id="amalgamation-panel">
```

Comment:
This (and the other alert components) is now a single expansion panel, instead of a list of panels.